salt/metalk8s/orchestrate/worker_plane.sls

```diff
@@ -0,0 +1,93 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+{% set node = pillar.get('node_name', 'node1') %}
+
+Run bootstrap prechecks:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls: metalk8s.bootstrap.precheck
+
+Run bootstrap kube client:
+  salt.state:
+    - saltenv: {{ saltenv }}
+    - tgt: {{ node }}
+    - sls: metalk8s.bootstrap.kubernetes_client
+    - pillar:
+        repo:
+          online_mode: false
+          local_mode: false
+          host: {{ control_plane_ip }}
+        registry_ip: {{ control_plane_ip }}
+
+Run bootstrap kubelet:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.bootstrap.kubelet_install
+    - pillar:
+        repo:
+          online_mode: false
```

Comment:
Why do these values need to be overridden?

Reply:
No specific reason, just a bad copy paste